### PR TITLE
docs: enhance some docs parts

### PR DESCRIPTION
**Pull request by 0xisk**
## Explanation

Enhancing the existing docs. 

## Related Issues

N/A

## Screenshots

N/A

## Manual Testing Steps

N/A

## Pre-Merge Checklist

- [x] PR template is filled out
- [x] Pre-commit and pre-push hook checks are passed
- [ ] E2E tests are passed locally
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [ ] PR is linked to the appropriate GitHub issue
- [ ] PR has been added to the appropriate release Milestone

> PR template source from [github.com/MetaMask](https://github.com/MetaMask)

### CODE REVIEW

File: docs/src/guides/integration/_intro.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+
+```ts
+// Example
+const { name, host } = await client.getConnectedIdentity()
+```
+
+* `getHostPermissions`: Retrieves the permissions for the specified host.
+* `setHostPermissions`: Sets the permissions for the specified host.
+* `createIdentity`: Creates a new identity and returns the result.
+* `createDummyRequest`: Creates a dummy request and returns the result.
```
Comment (0xmad) [inline docs/src/guides/integration/_intro.md:51]: We don't have this method and we need to keep this method list relevant.

File: docs/src/guides/integration/connect.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+    declare global {
+        interface Window {
+            zkpr: {
+                connect: () => any
+            }
+        }
+    }// Some code
+```
```
Comment (0xmad) [inline docs/src/guides/integration/connect.md:49]: Why do we need this internal code snippet?

File: docs/src/guides/integration/create-identity.md [hunk 3]
```diff
@@ -1,59 +1,35 @@
...
+   */
+  async connectIdentity({ host }: IConnectIdentityRequestArgs): Promise<void> {
+    await this.post({
+      method: RPCAction.CONNECT_IDENTITY_REQUEST,
+      payload: {
+        host,
+      },
+    });
+  }
+```
```
Comment (0xmad) [inline docs/src/guides/integration/create-identity.md:26]: The same here. Why do we need this code? How will it help to use provider methods?

File: docs/src/what-is-cryptkeeper/_intro.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+These injected providers allow applications to establish a connection with the CryptKeeper extension, enabling them to leverage its features and services. The package includes functions for initializing the CryptKeeper provider within the browser extension's InjectedScript and for connecting to **CryptKeeper** from an application.
+
+### Video overview
+
+#### YouTube: CryptKeeper Demo
+> [![Video](https://img.youtube.com/vi/8nrggpxbJu8/maxresdefault.jpg)](https://youtu.be/8nrggpxbJu8)
+
+#### YouTube: DevCon 6 "Crypt Keeper ZK Identity Wallet"
+> [![Video](https://img.youtube.com/vi/F5ic66_eNAs/maxresdefault.jpg)](https://www.youtube.com/watch?v=F5ic66_eNAs)
```
Comment (0xmad) [inline docs/src/what-is-cryptkeeper/_intro.md:23]: I think these links are outdated

**Review by 0xmad — CHANGES_REQUESTED** [review]
@0xisk thanks, there are some comments

File: docs/src/what-is-cryptkeeper/_intro.md (hunk 4 above)
Comment (0xisk) [reply docs/src/what-is-cryptkeeper/_intro.md:23]: I moved those to resources.

**Comment by 0xisk** [comment]
@0xmad thanks for your review. I moved the videos to resources under "Old Video Resources" title. And for the other comments they would be fixed in another PR because its dependent on what I am working on now which is doing enhancements to the provider functions. So we can get this PR merged so I can test the `deployDocs` workflow.

**Pull request merged**